client_plugin/drivers/shared/shared_driver.go

```diff
@@ -124,11 +124,12 @@ func NewVolumeDriver(cfg config.Config, mountDir string) *VolumeDriver {
 	}
 
 	// initialize built-in etcd cluster
-	d.kvStore = etcdops.NewKvStore(d.dockerOps)
-	if d.kvStore == nil {
+	etcdKVS := etcdops.NewKvStore(d.dockerOps)
```

Comment:
Why you need a new variable here, I think in the original code, when d.kvStore is nil, you also error out. Do I miss anything here?